insights/parsers/wc_proc_1_mountinfo.py

```diff
@@ -0,0 +1,45 @@
+"""
+WcProc1Mountinfo - Command ``/usr/bin/wc -l /proc/1/mountinfo``
+===============================================================
+
+Parser for parsing the output of command ``/usr/bin/wc -l /proc/1/mountinfo``.
+"""
+
+from insights import Parser, parser
+from insights.core.dr import SkipComponent
+from insights.parsers import ParseException
+from insights.specs import Specs
+
+
+@parser(Specs.wc_proc_1_mountinfo)
+class WcProc1Mountinfo(Parser):
+    """
+    Provides the line counts of file ``/proc/1/mountinfo`` by parsing the
+    output of command ``/usr/bin/wc -l /proc/1/mountinfo``.
+
+    Attributes:
+        line_count(int): the line counts of file ``/proc/1/mountinfo``
+
+    Typical content looks like::
+
+        37 /proc/1/mountinfo
+
+    Examples:
+        >>> type(wc_info)
+        <class 'insights.parsers.wc_proc_1_mountinfo.WcProc1Mountinfo'>
+        >>> wc_info.line_count
+        37
+
+    Raises:
+        insights.parsers.SkipComponent: if the command output is empty or
+        missing file.
```

Comment:
you should add one more tab indent in this line, to make sure the doc is built as expected.